src/components/views/dialogs/UnknownDeviceDialog.js

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+var React = require("react");
+var sdk = require('../../../index');
+var MatrixClientPeg = require("../../../MatrixClientPeg");
```

Comment:
this appears to be unused. And if it wasn't, I'd be asking you to pass in the `matrixClient` in a property, rather than adding more references to `MatrixClientPeg`.